lib/classes/ConfigSchemaHandler/resolveAjvValidate.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Ajv = require('ajv').default;
+const objectHash = require('object-hash');
+const path = require('path');
+const os = require('os');
+const standaloneCode = require('ajv/dist/standalone').default;
+const fs = require('fs');
+const requireFromString = require('require-from-string');
+const fse = require('fs-extra');
+const deepSortObjectByKey = require('../../utils/deepSortObjectByKey');
+const ensureExists = require('../../utils/ensureExists');
+
+const getValidate = async (schema) => {
+  const cacheDir = path.resolve(os.homedir(), '.serverless/schemas');
```

Comment:
This variable can be top level in module (it's not specific to passed in schema)